app/assets/javascripts/terrier/tables.coffee

```diff
@@ -8,11 +8,16 @@ _numberRegex = /^(?=.)([+-]?([0-9]*)(\.([0-9]+))?)$/g # pos/neg integer/float
 
 window.tables.initSortable = (ui = $(document), col = null, dir = null) ->
 	params = getUrlParams()
-	col = if col then col else params.sortable_col
-	dir = if dir then dir else params.sortable_dir
-	if col && dir
-		ui.find("table.sortable th a[data-column=#{col}]").each (_, link) ->
-			window.tables.sortByColLink($(link), col, dir)
+	path = window.location.pathname
+	[storedCol, storedDir] = window.localStorage.getItem "sortable:#{path}"
+		?.split(',') ? [null, null]
+
+	col = col ? params.sortable_col ? storedCol 
+	dir = dir ? params.sortable_dir ? storedDir
+	
+	return unless col && dir
```

Comment:
The persistence should be opt-in, and also it's possible that there are multiple sortable tables on the same page, so the pathname isn't a suitable storage key.

I think it would make sense to use a data attribute on the table element called `data-sort-persistence-key`, and if that is present, use persistence with the key in that data attribute. That way, you opt in by adding the data attribute with a key, and we eliminate the key uniqueness issue because each table can specify its own key.